package.json

```diff
@@ -82,7 +82,8 @@
     "build:ts": "tsc --build tsconfig.release.json",
     "build:ts:watch": "yarn build:ts --watch --preserveWatchOutput",
     "build": "yarn clean && yarn build:js && yarn build:ts && yarn copy-flowtypes",
-    "prepare": "yarn build"
+    "prepare": "yarn build",
+    "test:react:17": "scripts/test_react_17"
```

Comment:
Since we are RN TL maybe we should rather name our test script with `React Native version, e.g. `test-react-native-0.68.3`, @AugustinLF wdyt?

Reply:
BTW since we're invoking `test:ci` underneath, so maybe `test:ci:react-native-0.68.3`? Wdyt?

Reply:
We are RN TL, but I would argue that here we are actually testing the react differences, not as much the react-native ones (and we're still using react, I'd argue the problem is that `@testing-library/react` should be renamed `@testing-library/react-dom`^^')